static_src/components/route_form.jsx

```diff
@@ -155,6 +158,17 @@ export default class RouteForm extends React.Component {
             className={ this.styler('route-form-preview') }
             value={ this.fullUrl }
           ></input>
+          <div style={{ paddingLeft: '.75rem' }}>
+            {(() => {
+              if (this.props.error) {
+                return <FormError message={this.props.error.description} />;
```

Comment:
just a comment here and two lines below to indicate that this is for creation error display vs the other is for _RUD